src/Temperature-Accretech_UFseries/libs/accretech_uf.py

```diff
@@ -184,7 +186,8 @@ def __init__(self, port: pysweepme.Ports.Port):
 
     def __del__(self) -> None:
         """When the object is deleted, the SRQ event is unregistered."""
-        self.unregister_srq_event()
+        pass
+        # self.unregister_srq_event()
```

Comment:
Should better be deleted. It's not doing anything and the `__del__` function can be misleading. The separate register / unregister functions below are much better.